<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="nw45136" failures="6" tests="7" name="TestNG tests" time="0.244" errors="0" timestamp="18 Mar 2021 04:52:48 GMT">
  <testcase classname="banking.TransactionNGTest" name="testGetAdditionalCharges" time="0.002"/>
  <testcase classname="banking.TransactionNGTest" name="testGetAmount" time="0.001">
    <failure type="java.lang.AssertionError" message="The test case is a prototype.">
      <![CDATA[java.lang.AssertionError: The test case is a prototype.
	at banking.TransactionNGTest.testGetAmount(TransactionNGTest.java:88)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- testGetAmount -->
  <testcase classname="banking.TransactionNGTest" name="testGetStatus" time="0.0">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at banking.TransactionNGTest.testGetStatus(TransactionNGTest.java:101)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- testGetStatus -->
  <testcase classname="banking.TransactionNGTest" name="testGetTransactionTime" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [null] but found [2021-02-02T08:42:02]">
      <![CDATA[java.lang.AssertionError: expected [null] but found [2021-02-02T08:42:02]
	at banking.TransactionNGTest.testGetTransactionTime(TransactionNGTest.java:129)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- testGetTransactionTime -->
  <testcase classname="banking.TransactionNGTest" name="testGetTransactionType" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [] but found [DEPOSIT]">
      <![CDATA[java.lang.AssertionError: expected [] but found [DEPOSIT]
	at banking.TransactionNGTest.testGetTransactionType(TransactionNGTest.java:144)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- testGetTransactionType -->
  <testcase classname="banking.TransactionNGTest" name="testSetStatus" time="0.002">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at banking.TransactionNGTest.testSetStatus(TransactionNGTest.java:115)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- testSetStatus -->
  <testcase classname="banking.TransactionNGTest" name="testToString" time="0.096">
    <failure type="java.lang.AssertionError" message="expected [DEPOSIT             2021-02-02T08:42:02      1000.00         0.00                     SUCCESS   ] but found [[
DEPOSIT             2021-02-02T08:42:02      1000.00         0.00                     null      
]]">
      <![CDATA[java.lang.AssertionError: expected [DEPOSIT             2021-02-02T08:42:02      1000.00         0.00                     SUCCESS   ] but found [[
DEPOSIT             2021-02-02T08:42:02      1000.00         0.00                     null      
]]
	at banking.TransactionNGTest.testToString(TransactionNGTest.java:162)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- testToString -->
</testsuite> <!-- TestNG tests -->
